src/Element.cc

```diff
@@ -697,22 +744,54 @@ void Element::PrintValues(const std::string &_prefix,
                           bool _includeDefaultElements,
                           bool _includeDefaultAttributes,
                           const PrintConfig &_config) const
+{
+  std::ostringstream ss;
+  sdf::Errors errors;
+  PrintValuesImpl(_prefix,
+                  _includeDefaultElements,
+                  _includeDefaultAttributes,
+                  _config,
+                  ss,
+                  errors);
+  sdf::throwOrPrintErrors(errors);
+  std::cout << ss.str();
+}
+
+/////////////////////////////////////////////////
+void Element::PrintValues(sdf::Errors &_errors,
+                          const std::string &_prefix,
+                          bool _includeDefaultElements,
+                          bool _includeDefaultAttributes,
+                          const PrintConfig &_config) const
 {
   std::ostringstream ss;
   PrintValuesImpl(_prefix,
                   _includeDefaultElements,
                   _includeDefaultAttributes,
                   _config,
-                  ss);
+                  ss,
+                  _errors);
   std::cout << ss.str();
 }
 
 /////////////////////////////////////////////////
 std::string Element::ToString(const std::string &_prefix,
                               const PrintConfig &_config) const
+{
+  sdf::Errors errors;
+  std::ostringstream out;
+  this->ToString(out, errors, _prefix, true, false, _config);
```

Comment:
nit: This could be implemented using the `ToString` that takes an `_errors`.

Reply:
Changed in https://github.com/gazebosim/sdformat/pull/1141/commits/83e7072e9da47fb738772bc13a998f1bca60928d.